src/Command/CommandBase.php

```diff
@@ -1434,6 +1434,7 @@ protected function promptChooseCloudSite($cloud_environment) {
       $site = reset($sites);
       return $site;
     }
+    $this->io->writeln("This is a multisite application. Drupal will load the default site unless you've configured sites.php for this environment: https://docs.acquia.com/cloud-platform/develop/drupal/multisite/");
```

Comment:
We should really only write this text once. Throw it in a method and call it from both places.

Reply:
You can also consider using $this->io->warning(), but it's a bit alarming.

Reply:
Actually $this->io->note() is probably better. Looks like this:
![image](https://user-images.githubusercontent.com/539205/118174307-a5d0ff00-b3fc-11eb-87a6-70abec5adc53.png)